jsconfig.json

```diff
@@ -2,5 +2,10 @@
   "compilerOptions": {
     "baseUrl": "src"
   },
-  "include": ["src"]
+  "include": [
+    "src",
+    "SectionSearchbar/SectionSearch.jsx",
+    "SectionSearchbar/SearchBar",
+    "SectionSearchbar/RotateLogo"
```

Comment:
What does this do ?

Reply:
I don't know, it was added automatically...